packages/flutter_tools/flutter_tools.iml

```diff
@@ -30,16 +34,10 @@
       <excludeFolder url="file://$MODULE_DIR$/test/data/intellij/plugins/Dart/packages" />
       <excludeFolder url="file://$MODULE_DIR$/test/data/intellij/plugins/packages" />
       <excludeFolder url="file://$MODULE_DIR$/test/data/packages" />
+      <excludeFolder url="file://$MODULE_DIR$/test/ios/packages" />
       <excludeFolder url="file://$MODULE_DIR$/test/packages" />
-      <excludeFolder url="file://$MODULE_DIR$/test/replay/osx/packages" />
-      <excludeFolder url="file://$MODULE_DIR$/test/replay/osx/simulator_application_binary/file/packages" />
-      <excludeFolder url="file://$MODULE_DIR$/test/replay/osx/simulator_application_binary/packages" />
-      <excludeFolder url="file://$MODULE_DIR$/test/replay/osx/simulator_application_binary/platform/packages" />
-      <excludeFolder url="file://$MODULE_DIR$/test/replay/osx/simulator_application_binary/process/packages" />
-      <excludeFolder url="file://$MODULE_DIR$/test/replay/osx/simulator_application_binary/vmservice/packages" />
       <excludeFolder url="file://$MODULE_DIR$/test/replay/packages" />
-      <excludeFolder url="file://$MODULE_DIR$/test/src/base/packages" />
-      <excludeFolder url="file://$MODULE_DIR$/test/src/ios/packages" />
+      <excludeFolder url="file://$MODULE_DIR$/test/runner/packages" />
       <excludeFolder url="file://$MODULE_DIR$/test/src/packages" />
       <excludeFolder url="file://$MODULE_DIR$/tool/packages" />
     </content>
```

Comment:
They were, so I'd have less scm churn here. Note that the not-yet-released version of IntelliJ won't generate these excludes, so there won't be as many changes to these files.